Items/06_Explosives.txt

```diff
@@ -70,16 +73,18 @@ investigation checks.
 == ECSC Fragmentation Grenade $90 ==
 
 Damage: D10 Thermal Damage to exact spot of impact
-Ranged Miss Chance (to land where intended): 5 out to 10+[DEX or STR]*2 meters
+Miss Chance: 5
+Range: 10+[DEX or STR]*2 meters
 Armor Piercing Level: 1
 Reflex Modifier: -10
 Reflex Save: 100
 
-    Each Target within the blast radius takes multiple hits. 45 ballistic 
+    Each Target within the blast radius takes three shrapnel hits. 45 ballistic 
 damage per hit, automatically hits armor. Targets take full damage to the torso, 
 legs, and arms (if armor is pierced or not present): (3*45 damage)+2xBleed 
 within a 2.5m radius.Only legs and torso (2*45 damage)+1xBleed within a 5m 
-radius. At least one hit must damage health to cause Bleed.
+radius. At least one hit must damage health to cause Bleed. Recall that multiple 
```

Comment:
I like the changes here that make the damage more obvious. I think that there is still some ambiguity in the 'at least one hit causes bleed', does the player choose which wounds have the bleed tokens attached? Are the 2 tokens attached to one wound or are they one token per wound, but only 2? It would make them a little more powerful, but it would make more sense to me to have a bleed token for each shrapnel wound, just for simplicity's sake.

Reply:
I suggest phrasing like: the frag grenade applies 3 wounds, and if any damage health then the target gets 2 bleed tokens.